man/systemd.device.xml

```diff
@@ -152,9 +151,135 @@
+        </para></listitem>
+      </varlistentry>
+
+      <varlistentry>
+        <term><varname>IOCostModelRSeqIops=</varname></term>
+        <listitem><para> Maximum sequential read IOs per second. Takes a value between 0 and <constant>UINT64_MAX</constant>. Defaults to 0.
+        This controls the fourth field of the <literal>io.cost.model</literal> control group attribute. For details about this control group attribute,
+        see <ulink url="https://www.kernel.org/doc/html/latest/admin-guide/cgroup-v2.html#io-interface-files">IO Interface Files</ulink>
+        </para></listitem>
+      </varlistentry>
+
+      <varlistentry>
+        <term><varname>IOCostModelRRandIops=</varname></term>
+        <listitem><para> Maximum random read IOs per second. Takes a value between 0 and <constant>UINT64_MAX</constant>. Defaults to 0.
+        This controls the fifth field of the <literal>io.cost.model</literal> control group attribute. For details about this control group attribute,
+        see <ulink url="https://www.kernel.org/doc/html/latest/admin-guide/cgroup-v2.html#io-interface-files">IO Interface Files</ulink>
+        </para></listitem>
+      </varlistentry>
+
+      <varlistentry>
+        <term><varname>IOCostModelWbps=</varname></term>
+        <listitem><para> Maximum sequential write IO throughput in bytes per second. Takes a value between 0 and <constant>UINT64_MAX</constant>. Defaults to 0.
+        This controls the sixth field of the <literal>io.cost.model</literal> control group attribute. For details about this control group attribute,
+        see <ulink url="https://www.kernel.org/doc/html/latest/admin-guide/cgroup-v2.html#io-interface-files">IO Interface Files</ulink>
+        </para></listitem>
+      </varlistentry>
+
+      <varlistentry>
+        <term><varname>IOCostModelWSeqIops=</varname></term>
+        <listitem><para> Maximum sequential write IOs per second. Takes a value between 0 and <constant>UINT64_MAX</constant>. Defaults to 0.
+        This controls the seventh field of the <literal>io.cost.model</literal> control group attribute. For details about this control group attribute,
+        see <ulink url="https://www.kernel.org/doc/html/latest/admin-guide/cgroup-v2.html#io-interface-files">IO Interface Files</ulink>
+        </para></listitem>
+      </varlistentry>
+
+      <varlistentry>
+        <term><varname>IOCostModelWRandIops=</varname></term>
+        <listitem><para> Maximum random write IOs per second. Takes a value between 0 and <constant>UINT64_MAX</constant>. Defaults to 0.
+        This controls the eighth field of the <literal>io.cost.model</literal> control group attribute. For details about this control group attribute,
+        see <ulink url="https://www.kernel.org/doc/html/latest/admin-guide/cgroup-v2.html#io-interface-files">IO Interface Files</ulink>
+        </para></listitem>
+      </varlistentry>
+    </variablelist>
```

Comment:
I must say I am not too fond of explaining so little about the settings and immediately pointing to the kernel docs. Pointing to the kernel docs is OK, but it should be in style "for further details, see", and the documentation we carry right here should give you enough of an idea to not actually go in the kernel docs.

the way the man pages read right now, they are pretty useless on their own, i *have* to go to the kernel docs, and all the man pages help you with is identifying which field maps to which...